### ISSUE TITLE
OSX client is built without shibboleth support

### ISSUE THREAD

**Issue by yama6a** [nextcloud/client_theming#226]
The nextCloud OSX client is built without shibboleth support (DNO_SHIBBOLETH=1)
https://github.com/nextcloud/client_theming/blob/master/osx/build.sh#L31

This results in not being able to use e.g. Google SSO via SAML on OSX.

For reference:
owncloud/client#6223

Bug or feature?

**Comment by rullzer** [comment]
Yes. But this is done since we build with Qt5.9 on OSX. Which does not support webkit.
So there is no choice currently. We are working on getting support for this back.
